pkg/acsengine/engine.go

```diff
@@ -850,7 +850,7 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) map[str
 				case "kubeClusterCidr":
 					val = "10.244.0.0/16"
 				case "kubeBinariesVersion":
-					val = string(api.KubernetesLatest)
+					val = string(api.KubernetesCurrent)
```

Comment:
If you're going to change this, I'd opt for something clearer like `KubernetesDefaultVersion`, or just not change it in this PR.